src/test/versionbits_tests.cpp

```diff
@@ -14,28 +14,48 @@
 /* Define a virtual block time, one block per 10 minutes after Nov 14 2014, 0:55:36am */
 static int32_t TestTime(int nHeight) { return 1415926536 + 600 * nHeight; }
 
-static const Consensus::Params paramsDummy = Consensus::Params();
+static constexpr const char* StateName(ThresholdState state) {
+    return (state == ThresholdState::DEFINED ?     "DEFINED" :
+            state == ThresholdState::STARTED ?     "STARTED" :
+            state == ThresholdState::LOCKED_IN ?   "LOCKED_IN" :
+            state == ThresholdState::ACTIVE ?      "ACTIVE" :
+            state == ThresholdState::FAILED ?      "FAILED" :
+            "");
```

Comment:
Couldn't this use a `switch` statement?

Reply:
This PR is intended to merge cleanly to 0.21, which only requires C++11. (`switch` inside `constexpr` functions requires C++14)